api/middleware/authorization.ts

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken';
+require('dotenv').config();
+
+module.exports = async (req: any, res: any, next: any) => {

```

Comment:
To trzeba bedzie poprawić w całym kodzie, więc może być chociaż fajnie jakbyśmy to otypowali